languages/go/tests/enforcement_test.go

```diff
@@ -196,6 +213,35 @@ func TestAuthorizedActions(t *testing.T) {
 	}
 }
 
+type TestAdapter struct {
+}
+
+func (a TestAdapter) BuildQuery(filter *types.Filter) (interface{}, error) {
+	return nil, nil
+}
+
+func (a TestAdapter) ExecuteQuery(query interface{}) (interface{}, error) {
+	return nil, nil
+}
+
+func TestAuthorizedQuery(t *testing.T) {
+	o := getOso(t)
+	var err error
+
+	o.SetDataFilteringAdapter(&TestAdapter{})
+
+	o.LoadString("allow(_actor: User, \"get\", resource: Widget) if resource.Parent.Id = 1;")
+
+	actor := User{Name: "Sally"}
+	resource := Widget{Id: 1}
+	_, _ = o.IsAllowed(actor, "get", resource)
+
+	results, err := o.AuthorizedQuery(actor, "get", "Widget")
+	fmt.Printf("%v\n", results)
```

Comment:
Is it possible to add a test based on https://github.com/osohq/oso-go-df-sqlboiler ?

Reply:
Not really, without adding sqlboiler as a dependency (as far as I can tell?)